src/components/Patient/LinkDepartmentsSheet.tsx

```diff
@@ -117,12 +117,14 @@ function DeleteOrganizationButton({
   entityId,
   facilityId,
   onSuccess,
+  dataCy,
```

Comment:
Instead of passing it as props, consider adding it directly, also even if it is passed as props, make it optional